manifests/init.pp

```diff
@@ -387,6 +405,18 @@
     }
   }
 
+  $sources.each |$key, $value| {
+    apt::source { $key:
+      * => $value,
+    }
+  }
```

Comment:
This look like a duplicate of the above block.

Reply:
you're right, I removed the block